packages/app/src/cli/services/fetch-dashboard-extensions.ts

```diff
@@ -1,6 +1,6 @@
 import {DeveloperPlatformClient} from '../utilities/developer-platform-client.js'
 
-export async function getActiveDashboardExtensions({
```

Comment:
This file should definitely be renamed to `fetch-extensions` or something that doesn't have `dashboard` in it